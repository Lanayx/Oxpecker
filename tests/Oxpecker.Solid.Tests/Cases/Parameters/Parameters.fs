module Oxpecker.Solid.Tests.Cases.Parameters

open Oxpecker.Solid
open Oxpecker.Solid.Aria

[<SolidComponent>]
let Test (id: int) =
    let hello = "Hello "
    div(id=string id, class'="testclass", ariaLabelledBy="testlabel") {
        Fragment() { hello }; "world!"
    }
