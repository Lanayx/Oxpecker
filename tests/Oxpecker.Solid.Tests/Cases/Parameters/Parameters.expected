import { int32ToString } from "./fable_modules/fable-library-js.4.21.0/Util.js";

export function Test(id) {
    return <div id={int32ToString(id)}
        class="testclass"
        aria-labelledby="testlabel">
        <>
            Hello 
        </>
        world!
    </div>;
}

